### PR TITLE
make openstack kops-controller boostrap auth better

### PR DESCRIPTION
**Pull request by zetaab**

### CODE REVIEW

File: nodeup/pkg/bootstrap/install.go [hunk 1]
```diff
@@ -161,6 +161,7 @@ func (i *Installation) buildSystemdJob() *nodetasks.InstallService {
 	manifest := &systemd.Manifest{}
 	manifest.Set("Unit", "Description", "Run kOps bootstrap (nodeup)")
 	manifest.Set("Unit", "Documentation", "https://github.com/kubernetes/kops")
+	manifest.Set("Unit", "ConditionPathExists", "!/srv/kubernetes/kubelet-server.crt")
```
Comment (zetaab) [inline nodeup/pkg/bootstrap/install.go:164]: if I understood correctly we cannot use this in AWS? Is there easy way how we could pass cloud information to this package @olemarkus

**Comment by zetaab** [comment]
@johngmyers @hakman could you check this? Currently in master (and in 1.26) we do not have bullet proof version

File: upup/pkg/fi/cloudup/openstack/verifier.go [hunk 2]
```diff
@@ -75,12 +82,42 @@ func NewOpenstackVerifier(opt *OpenStackVerifierOptions) (bootstrap.Verifier, er
...
+	}
+	return kubernetes.NewForConfig(config)
+}
+
+// readKubeConfig ...
+func readKubeConfig() (*restclient.Config, error) {
+	return clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{}).ClientConfig()
+}
```
Comment (hakman) [inline upup/pkg/fi/cloudup/openstack/verifier.go:118]: This should be inlined in `newClientSet()`.

**Review by hakman — CHANGES_REQUESTED** [review]
/lgtm

**Pull request merged**